types/testing-library__jest-dom/test/testing-library__jest-dom-global-tests.ts

```diff
@@ -22,6 +22,7 @@ expect(element).toHaveClass('cls1', 'cls2', 'cls3', 'cls4');
 expect(element).toHaveFocus();
 expect(element).toHaveFormValues({ foo: 'bar', baz: 1 });
 expect(element).toHaveStyle('display: block');
+expect(element).toHaveStyle({ display: 'block' });
```

Comment:
Minor thing, could you add another property that uses a different type, like `width: 100` (just to verify that non-string values are also accepted)?